corehq/ex-submodules/casexml/apps/phone/caselogic.py

```diff
@@ -547,6 +556,19 @@ def __repr__(self):
         )
 
 
+class UserCaseSyncCouchBatch(CaseSyncCouchBatch):
+    def __init__(self, user, global_state, domain, last_sync, chunksize, case_sharing=False):
+        super(UserCaseSyncCouchBatch, self).__init__(
+            global_state, domain, last_sync, chunksize, None, case_sharing=case_sharing)
+        self.user = user  # phone.models.User instance
+
+    def _actual_owned_cases(self):
+        from corehq.apps.hqcase.utils import get_case_by_domain_hq_user_id
+
+        usercase = get_case_by_domain_hq_user_id(self.domain, self.user.user_id, include_docs=False)
```

Comment:
I think you should just fetch the whole doc unless you're going to do something similar to here: https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/casexml/apps/phone/caselogic.py#L509-509